spec/dsl_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+RSpec.describe "Pundit RSpec DSL" do
+  before do
+    RSpec.configure do |config|
+      config.include ::Pundit::RSpec::DSL
+    end
+  end
+
+  let(:block) { proc { "block content" } }
+
+  it "calls describe with the correct metadata and without :focus" do
+    expected_metadata = { permissions: %i[:item1, :item2], caller: instance_of(Array) }
+    expect(self).to receive(:describe).with("item1 and item2", hash_including(expected_metadata)) do |&block|
```

Comment:
Careful here, `hash_including` is lenient when it comes to _additional_ content. If the implementation always added `focus: true` then this test would still pass, and so would the other one.

Instead of `hash_including` we want `match` here instead.